src/lib.rs

```diff
@@ -259,14 +259,13 @@ impl EguiMq {
     /// Call from your [`miniquad::EventHandler`].
     pub fn mouse_wheel_event(&mut self, dx: f32, dy: f32) {
         let delta = egui::vec2(dx, dy);
+        let modifiers = self.egui_input.modifiers;
 
-        let event = if self.egui_input.modifiers.ctrl {
-            // Treat as zoom instead:
-            egui::Event::Zoom((delta.y / 200.0).exp())
-        } else {
-            egui::Event::Scroll(delta)
-        };
-        self.egui_input.events.push(event);
+        self.egui_input.events.push(egui::Event::MouseWheel {
+            modifiers,
+            unit: egui::MouseWheelUnit::Line,
```

Comment:
Is this really then unit miniquad uses? Sadly it is not documented: https://docs.rs/miniquad/latest/src/miniquad/event.rs.html#170

Maybe @not-fl3 knows?

Reply:
Yes, I also noticed that it's not documented there.

I ended up doing some testing and adding `dbg!(delta)` inside the event handler, which ended up printing multiples of one every time I scrolled:  `[1.0, 0.0], [-1.0, 0.0], [0.0, 1.0]` (every single event was `+- 1.0`).  I tested using both my mouse and trackpad, and got a consistent `[1.0, 0.0]` for every  'click' of the scroll wheel, so I'm, fairly confident it's in lines.

Reply:
I don't :| 
On most platforms its just a raw data from an OS message, on some its fixed 1.0/-1.0.
We do not normalize it in any way.